ccd-definition/CaseTypeTab.json

```diff
@@ -48,7 +48,17 @@
     "TabLabel": "Draft orders",
     "TabDisplayOrder": 3,
     "CaseFieldID": "caseManagementOrder",
-    "TabFieldDisplayOrder": 1
+    "TabFieldDisplayOrder": 2
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "Channel": "CaseWorker",
+    "TabID": "DraftOrdersCmoTab",
+    "TabLabel": "Draft orders",
+    "TabDisplayOrder": 3,
+    "CaseFieldID": "allParties",
```

Comment:
you cant persist allParties - those need to be copied to the order object itself please follow how this is done in SDO.

Reply:
Ah ofc, this is not even needed!